examples/dmrg/write_to_disk/1d_heisenberg.jl

```diff
@@ -0,0 +1,44 @@
+using ITensors
+using ITensors.Strided
+using LinearAlgebra
+using Printf
+using Random
+
+Random.seed!(1234)
+BLAS.set_num_threads(1)
+Strided.set_num_threads(1)
```

Comment:
Are these settings necessary for doing write-to-disk DMRG? Or just recommended say?

Reply:
Just copied from another example that was using block sparse multithreading. Though generally it is recommended to turn of Strided threading when using BLAS threading (which is only needed if Julia is started with multiple threads).

Reply:
So no, this isn't anything particular to write-to-disk.

Reply:
Thanks - I saw that you removed it in a recent commit. Agreed, since I wouldn't want a beginning user to think these lines are necessary for this example.